lib/streamlit/components/v1/components.py

```diff
@@ -109,6 +108,33 @@ def create_instance(
         if len(args) > 0:
             raise MarshallComponentException(f"Argument '{args[0]}' needs a label")
 
+        try:
+            import pyarrow
+            from streamlit.elements import arrow_table
+        except ImportError:
+            import sys
+
+            if sys.version_info >= (3, 9):
+                raise StreamlitAPIException(
+                    """To use Custom Components in Streamlit, you need to install
+PyArrow. Unfortunately, PyArrow does not yet support Python 3.9.
+
+You can either switch to Python 3.8 with an environment manager like PyEnv, or
+[install Streamlit with conda](https://discuss.streamlit.io/t/note-installation-issues-with-python-3-9-and-streamlit/6946/7):
+
+`conda install -c conda-forge streamlit`
```

Comment:
I would just add another line after the conda install bit for visuals
```suggestion
`conda install -c conda-forge streamlit`

```

![image](https://user-images.githubusercontent.com/24946400/102379942-61e4e580-3f95-11eb-9a4d-351fdde385e5.png)